site/content/contribute/more-info/server/developer-setup/arch.md

```diff
@@ -71,7 +71,7 @@
     {"AndroidLatestVersion":"","AndroidMinVersion":"","DesktopLatestVersion":"","DesktopMinVersion":"","IosLatestVersion":"","IosMinVersion":"","status":"OK"}
     ```
 
-    **Note:** Browsing directly to http://localhost:8065/ will display a `404 Not Found` until the web app is configured. See [Web App Developer Setup]({{< ref "/contribute/more-info/webapp/developer-setup" >}}) and [Mobile App Developer Setup]({{< ref "/contribute/more-info/mobile/developer-setup" >}}) for additional setup.
+    **Note:** Browsing directly to http://localhost:8065/ will display a `404 Not Found` until the web app is configured. See [Web App Developer Setup](https://developers.mattermost.com/contribute/more-info/webapp/developer-setup/) and [Mobile App Developer Setup](https://developers.mattermost.com/contribute/more-info/mobile/developer-setup/) for additional setup.
```

Comment:
Please revert these changes; we use relative links in the developer docs and we would like to keep them consistent.

Reply:
:ok_hand: Reverted all relative links changes in 2d88ef36fbb69e919fee7a9da850b6f2347c41db. Let me know if it's ok, I will do necessary fixup/rebase to clean-up the git tree of this PR.

Reply:
Rebase and fixup done.